src/views/__tests__/TransactionSigner.test.tsx

```diff
@@ -6,32 +6,39 @@ import { fireEvent, waitFor, screen } from "@testing-library/react";
 import { render } from "helpers/testHelpers";
 import { TransactionSigner } from "views/TransactionSigner";
 import * as transactionSignerActions from "actions/transactionSigner";
+import { MOCK_SIGNED_TRANSACTION } from "./__mocks__/transactions";
 
 jest.mock("@stellar/freighter-api", () => ({
   isConnected: () => new Promise((resolve) => resolve(true)),
-  signTransaction: () => new Promise((resolve) => resolve(SIGNED_TRANSACTION)),
+  signTransaction: () =>
+    new Promise((resolve) => resolve(MOCK_SIGNED_TRANSACTION)),
 }));
 
 jest.mock("@albedo-link/intent", () => ({
   tx: () =>
     new Promise((resolve) =>
-      resolve({ signed_envelope_xdr: SIGNED_TRANSACTION }),
+      resolve({ signed_envelope_xdr: MOCK_SIGNED_TRANSACTION }),
     ),
 }));
 
+const SOURCE_KEYPAIR = {
+  publicKey: "GCQ5VP5V7JP3SCLPZVNWINFWDEY4YDQ2UVI2BSNSGENLPKNTUMK42QUN",
+  secretKey: "SDBJANOEQHXOC2LWPP2CVYUJK4VR7MXS2ONKKSHHADOCEBAPOGTT4WHP",
+};
+const SIGNER_KEYPAIR = {
+  publicKey: "GDR34X73SODUQJYO4E7IFWPRGGVQMLJFEWISAAIMEEX3LOGBFCDO735C",
+  secretKey: "SA6QM7HQFTQGI5FRQJVP2Q2URI4W2SIEWPUY2XFDAV4BQD3L5JWJJRQN",
+};
```

Comment:
Testnet Reset wiped out the test acct's I had setup for tx signer :(

Making this a bit more explicit so it's easier to restore these accts if needed in the future